doc/api/errors.md

```diff
@@ -459,10 +459,27 @@ Returns a number corresponding to the **negated** error code, which may be
 referenced in `man 2 intro`. For example, an `ENOENT` error has an `errno` of
 `-2` because the error code for `ENOENT` is `2`.
 
+In some mudules, returns the same string as `error.code`.
+
 #### error.syscall
 
 Returns a string describing the [syscall][] that failed.
 
+#### error.path
+
+Returns a string of the specified invalid pathname in case of using some modules
```

Comment:
@mscdex @Trott @Qard Thanks for your review.
I did s/Returns/Is/ about this page's properties.